pig/src/main/java/org/jboss/pnc/bacon/pig/impl/config/PigConfiguration.java

```diff
@@ -79,6 +79,8 @@ public class PigConfiguration implements Validate {
     private Map<String, Map<String, ?>> addons = new HashMap<>();
     private String releaseStorageUrl;
 
+    private String temporaryBuildAlignmentPreference;
```

Comment:
I presume this denotes the yaml config value - what happens if they set it to something other than temporary or persistent?